ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -624,6 +624,15 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= []
       }
     )
+  ; ( "pool-configure-repository-proxy"
+    , {
+        reqd= ["proxy-url"]
+      ; optn= ["proxy-username"; "proxy-password"]
+      ; help= "Configure proxy used in syncing with the enabled repositories"
```

Comment:
The grammar of "used in syncing" feels awkward to me. I'd say: 

Configure proxy for RPM package repositories.